asm/macros/scrcmd.inc

```diff
@@ -1094,26 +1095,26 @@
     .short 181
     .endm
 
-    .macro ScrCmd_0B6 arg0
+    .macro ScrCmd_0B6 approachNum
     .short 182
-    .short \arg0
+    .short \approachNum
     .endm
 
-    .macro ScrCmd_0B7 arg0, arg1
+    .macro CheckIsTrainerTaskDone approachNum, destVar
     .short 183
-    .short \arg0
-    .short \arg1
+    .short \approachNum
+    .short \destVar
     .endm
 
-    .macro ScrCmd_0B8 arg0
+    .macro GetTrainerType destVar
```

Comment:
Let's keep this unnamed for now. The value this command gets is not actually a trainer type (as per `generated/trainer_types.txt`).